conf/machine/raspberrypi0-2w-64.conf

```diff
@@ -0,0 +1,21 @@
+#@TYPE: Machine
+#@NAME: RaspberryPi0 2 Wifi Development Board
+#@DESCRIPTION: Machine configuration for the RaspberryPi0 2 Wifi in 64 bits mode
+
+MACHINEOVERRIDES = "raspberrypi3-64:${MACHINE}"
```

Comment:
Shall we define this after the include? Just to be safe.

Reply:
done